### PR TITLE
updated RFC

### PR DESCRIPTION
**Pull request by jmazanec15**
*Issue #, if available:*
#48 

*Description of changes:*
Updated RFC to incorporate latest code changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: RFC.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
 ## Proposed Solution
 
-Open Distro for Elasticsearch enables you to run nearest neighbor search on billions of documents across thousands of dimensions with the same ease as running any regular Elasticsearch query. You can use aggregations and filter clauses to further refine your similarity search operations. Power use-cases include product recommendations, fraud detection, image and video search, related document search, and more.
+The ODFE KNN plugin enables you to run an Approximate K Nearest Neighbor search on billions of documents across thousands of dimensions with the added usability and reliability of Elasticsearch. You can use aggregations and filter clauses to further refine your similarity search operations. Powerful use-cases include product recommendations, fraud detection, image and video search, related document search, and more.
```
Comment (vamshin) [inline RFC.md]: Elaborate ODFE?

File: RFC.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+The purpose of this document is to introduce a new Open Distro for Elasticsearch KNN plugin and to request comments from the community (RFC). This RFC covers only the high-level functionality of the KNN plugin and does not cover architecture and implementation details.
 
 ## Problem Statement
-Customers with similarity-based search use cases such as face search, fraud detection and recommendation systems have a need for finding K-Nearest Neighbor (KNN) using the machine learning models. The existing solutions for KNN search were inefficient and did not operate well at scale. Brute-force KNN search using CPU is very computationally intensive and generally too slow for large-scale real-time search applications.
+Customers with similarity-based search use cases like facial recognition, fraud detection and recommendation systems have a need for finding the K-Nearest Neighbor (KNN) of vectors in high dimensional space. The existing solutions for KNN search in distributed systems either do not operate well at scale or are difficult to use and integrate with current systems.
```
Comment (vamshin) [inline RFC.md]: "The existing solutions for KNN search do not operate well at scale, difficult to use, computationally intensive and generally too slow for large-scale real-time search applications." ?

**Review by vamshin — COMMENTED** [review]
Minor comments. Looks great

File: RFC.md (hunk 1 above)
Comment (jmazanec15) [reply RFC.md]: Will do

File: RFC.md (hunk 2 above)
Comment (jmazanec15) [reply RFC.md]: Thanks addressed this

**Review by vamshin — APPROVED** [review]

**Pull request merged**